Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,122 @@
+
+                Console.WriteLine($"{firstName} {surname} is {age} years old, his weight is {weight} kg and his height is {height} meters.\n" +
+                    $"Body-mass index (BMI) is: {bmi.ToString("0.00")}");
+
+                var answer = PromptString("\nDo you want to check BMI for another person? y/n: ").ToLowerInvariant();
+                doAnotherPerson = answer.Equals("y") || answer.Equals("yes");
+            }
+        }
+
+        public static string GetUserInput(string message)
+        {
+            Console.WriteLine(message);
+            return Console.ReadLine();
+        }
+
+        public static string PromptString(string message)
+        {
+            var userInput = GetUserInput(message);
+            
+            if (string.IsNullOrEmpty(userInput))
+            {
+                Console.Write("Name cannot be empty.");
+                return "-";
+            }
+
+            return userInput;
+        }
+
+        public static int PromptInt(string message)
+        {
+            var userInput = GetUserInput(message);
+
+            if (string.IsNullOrEmpty(userInput))
+            {
+                return 0;
+            }
+
+            bool isNotValidInt = !Int32.TryParse(userInput, out int number);
+            if (isNotValidInt)
+            {
+                Console.Write($"\"{userInput}\" is not a valid number.");
+                return -1;
+            }
+            return number;
+        }
+        public static float PromptFloat(string message)
+        {
+            var userInput = GetUserInput(message);
+
+            if (string.IsNullOrEmpty(userInput))
+            {
+                return 0f;
+            }
+
+            bool isNotValidFloat = !float.TryParse(userInput, out float number);
+            if (isNotValidFloat)
+            {
+                Console.Write($"\"{userInput}\" is not a valid number.");
+                return -1;
+            }
+            return number;
+        }
+        public static float CalculateBmi(float weight, float height)
+        {
+            if (weight <= 0 && height <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine($"Weight cannot be equal or less than zero, but was {weight}.");
+                Console.WriteLine($"Height cannot be less than zero, but was {height}.");
+                return -1;
+            }
+            if (weight <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine($"Weight cannot be equal or less than zero, but was {weight}.");
+                return -1;
+            }
+
+            if(height <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine($"Height cannot be equal or less than zero, but was {height}.");
+                return -1;
+            }
+
+            if(weight == height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not check floating point equality with exact values, use a range instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885729)